### PR TITLE
Add nebari-dask recipe

### PR DESCRIPTION
**Pull request by viniciusdc**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/nebari-dask/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+test:
+  imports:
+    - dask
+
+about:
+  home: https://github.com/quansight/qhub
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: "Qhub dask dependencies"
```
Comment (viniciusdc) [inline recipes/nebari-dask/meta.yaml:27]:
```suggestion
about:
  home: https://github.com/nebari-dev
  license: BSD-3-Clause
  license_family: BSD
  license_file: LICENSE
  summary: "Nebari dask dependencies."
```

File: recipes/nebari-dask/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+about:
+  home: https://github.com/nebari-dev
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: "Nebari dask dependencies."
+
+extra:
+  recipe-maintainers:
+    - costrouc
```
Comment (viniciusdc) [inline recipes/nebari-dask/meta.yaml:31]:
```suggestion
```

**Comment by viniciusdc** [comment]
I am removing @costrouc from the maintainer's list for now to get this merged. Once we do the next release, I can add again :smile:

**Comment by iameskild** [comment]
@viniciusdc I agree to help maintain this repo :)

**Comment by nebari-sensei** [comment]
@viniciusdc  I agree to help maintain this repo

**Comment by viniciusdc** [comment]
This is a renaming of [`qhub-dask`](https://github.com/conda-forge/qhub-dask-feedstock)

**Comment by viniciusdc** [comment]
@conda-forge/python

File: recipes/nebari-dask/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: "Nebari dask dependencies."
+
+extra:
+  recipe-maintainers:
+    - viniciusdc
+    - iameskild
+    - nebari-sensei
```
Comment (xhochy) [inline recipes/nebari-dask/meta.yaml:33]: According to the description, this is a bot account. Not sure what our policy on that is.

Comment (viniciusdc) [reply recipes/nebari-dask/meta.yaml:33]: I can remove that... but I would like to know if its possible to keep a team in the feedstock repo in the maintainer list.. would that work?

Comment (trallard) [reply recipes/nebari-dask/meta.yaml:33]: If we need more folks - you can add me @viniciusdc

Comment (viniciusdc) [inline recipes/nebari-dask/meta.yaml:33]:
```suggestion
```

Comment (viniciusdc) [inline recipes/nebari-dask/meta.yaml:33]:
```suggestion
    - trallard
```

**Comment by viniciusdc** [comment]
Hi @beckermr, could you have a look at this? Also, as @xhochy commented, should we mention in the conda-forge docs about bot's accounts as maintainers?

**Review by xhochy — APPROVED** [review]

**Comment by xhochy** [comment]
> Hi @beckermr, could you have a look at this as well? Also, as @xhochy commented, should we mention in the conda-forge docs about bot's accounts as maintainers?

Having the account is fine, you can re-add it on the feedstock. Sorry but wanted to make sure it's OK.

**Comment by viniciusdc** [comment]
> > Hi @beckermr, could you have a look at this as well? Also, as @xhochy commented, should we mention in the conda-forge docs about bot's accounts as maintainers?
> 
> Having the account is fine, you can re-add it on the feedstock. Sorry but wanted to make sure it's OK.

Thanks, @xhochy, no worries, I am glad you pointed that out :smile:

**Pull request merged**